addons/website/views/snippets/snippets.xml

```diff
@@ -1638,6 +1638,7 @@
         <we-input string="Border Width" class="o_we_sublevel_1" data-customize-website-variable=""
             data-variable="btn-secondary-outline-border-width" data-dependencies="btn_secondary_outline_opt"
             data-unit="px" data-save-unit="rem"/>
+        <we-fontfamilypicker string="Font Family" data-variable="buttons-font"/>
```

Comment:
The string is "Font family", with a lower case "f" in family, or else it is not translated like the other Font family options :wink: (note that it will still be displayed with a capital F in the editor :wink: )